sos/plugins/cifs.py

```diff
@@ -0,0 +1,30 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Cifs(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Cifs filesystem
+    """
+    packages = ('cifs-utils',)
```

Comment:
This plugin does not use anything from this package at all. Either get rid of the package trigger, and use a file trigger that is known to exist if this plugin should run (e.g. `/proc/fs/cifs`), or add something that this plugin does actually depend on.

Reply:
And looking at this entire directory hierarchy, it's tiny: there's no reason to not collect it all in the filesys plugin.